### DISCUSSION TITLE
How to get dark mode

### DISCUSSION THREAD

**Discussion by Rocky-25** [microsoft/fluentui-blazor#1526] [Q&A]
I'm using another library for some charts/graphs and it has allows a parameter to be passed for dark mode. I'm really struggling to get this out from FluentUI.

I've added this to the page I need to know 
<FluentDesignTheme StorageName="theme" @bind-Mode=Mode/>

And in the code have this

public DesignThemeModes Mode { get; set; }

 protected override async Task OnInitializedAsync()
 {
     Console.WriteLine($"Mode: {Mode}");
}

But it always prints "Mode: System", regards of being in dark or light mode. What am I doing wrong? How do I get this value correctly?

Thanks!

**Comment by vnbaaij** [comment]
Take a look at how we do it in the sample site. Source is in examples/Demo/Shared folder
